clients/deltalake/src/test/java/org/projectnessie/deltalake/ITDeltaLogBranches.java

```diff
@@ -98,7 +99,12 @@ void testCheckpoint() throws NessieNotFoundException {
     Assertions.assertEquals(64, expectedSize);
 
     String tableName = tempPath.getAbsolutePath() + "/_delta_log";
-    ContentKey key = ContentKey.of(tableName.split("/"));
+    // we can't simply do a tableName.split("/") as that would result with
+    // "/tmp/a/b/c in a string array where the first element is an empty string,
+    // which isn't supported by ContentKey
+    String[] elements =
+        StringUtils.stripEnd(StringUtils.stripStart(tableName, "/"), "/").split("/");
+    ContentKey key = ContentKey.of(elements);
```

Comment:
nit: its odd to have such a technical detail in the code of a single test... we should at least have a helper method with a proper name?